docs/reference/modules/network.asciidoc

```diff
@@ -80,6 +80,9 @@ The following special values may be passed to `network.host`:
 `_global_`::
 
   Any globally-scoped addresses on the system, for example `8.8.8.8`.
+  
+  `0`::
```

Comment:
The fact that `0` is recognized at all is due to [IP address normalization in OpenJDK](http://hg.openjdk.java.net/jdk/jdk/file/190b77982361/src/java.base/share/classes/sun/net/util/IPAddressUtil.java#l41) and I don't think that we should document that behavior as a "special value". Special values documented here are meant as "specially treated by Elasticsearch itself", i.e. those have a  [specific handling in the `NetworkService`](https://github.com/elastic/elasticsearch/blob/a9641960e1be101c0bcd4e9bbd28c3a2a2cabeb5/server/src/main/java/org/elasticsearch/common/network/NetworkService.java#L209-L226).